BUILD

```diff
@@ -0,0 +1,35 @@
+cc_library(
```

Comment:
Please add copyright headers to these new files

Reply:
Sure, can you point me to what I should copy-paste in?

Reply:
See any other file, substituting the line comment characters if necessary 
Thanks!